tests/core/vm/test_interrupt.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+
+from eth_hash.auto import keccak
+from eth_utils import (
+    int_to_big_endian,
+)
+
+from eth.vm.interrupt import (
+    MissingAccountTrieNode,
+    MissingBytecode,
+    MissingStorageTrieNode,
+)
+
+
+@pytest.fixture
+def address_with_balance():
+    return b'1' * 20
+
+
+@pytest.fixture
+def address_with_balance_hash(address_with_balance):
+    return keccak(address_with_balance)
+
+
+@pytest.fixture
+def balance():
+    return 10**18
+
+
+@pytest.fixture
+def address_with_bytecode():
+    return b'2' * 20
+
+
+@pytest.fixture
+def bytecode():
+    return b'aoeu'
+
+
+@pytest.fixture
+def bytecode_hash(bytecode):
+    return keccak(bytecode)
+
+
+@pytest.fixture
+def address_with_storage():
+    return b'3' * 20
```

Comment:
Note-to-self, we really should implement `evm.tools.factories` and make some `factory-boy` factories for these things.  I think it would help readability a lot.